fab/environments.yml

```diff
@@ -165,40 +165,45 @@ icds:
         num_processes: 1
       CaseSearchToElasticsearchPillow:
```

Comment:
We should just remove case search pillow. I don't htink it will ever be used on icds